app/models/config.rb

```diff
@@ -70,4 +95,100 @@ def self.start_day
     start ||= "monday"
     start.downcase.to_sym
   end
+
+  private
```

Comment:
My only concern here is that there isn't a 1:1 mapping of cleaners/validators to fields. For example, `start of week` and `hide practical support` both use `fix_capitalization` as their cleaner, and all the URLs share cleaners and validators. If, in the future, we need more granular control, we can always create field-specific methods and then this may make more sense? Or are you thinking: move all the generic functions up top, and then the field-specific?

Reply:
yeah, the latter!